css/style.css

```diff
@@ -0,0 +1,1155 @@
+      url("../assets/bg-patterns/6.png"),
+      url('../assets/images/hero-bg.png');
+    background-position: 0% 29%, 84% 90%, 96% 58%, 8% 94%, top center;
+    background-size: 40px, 80px, 30px, 40px, contain;
+  }
+
+  .hero__wrapper {
+    padding-bottom: 6rem;
+  }
+
+  .hero__page-headline {
+    font-size: var(--fs-950);
+    line-height: 4rem;
+  }
+
+  .hero__tagline {
+    font-size: var(--fs-800);
+  }
+
+  .hero__summit-info__dates {
+    font-size: var(--fs-800);
+  }
+
+  .hero__intro,
+  .about__intro {
+    padding: 2rem;
+    font-size: var(--fs-600);
+  }
+
+  .hero__contact-us {
+    font-size: var(--fs-600);
+  }
+}
+
+/* ============================================================ */
+
+/* Main Program Section Styles */
+.main-program {
+  background-color: var(--dark-blue-800);
+  background-image: url('../assets/decorative-elements/cross-pattern-1.svg');
+  background-size: 3.5vw;
+  color: var(--off-white);
+  padding: 4rem 0;
+}
+
+.main-program__headline {
+  font-size: var(--fs-700);
+  text-align: center;
+}
+
+.main-program__cards-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 0.5rem;
+  margin: 0;
+  margin-top: 2rem;
+  padding: 0;
+}
+
+.main-program__card {
+  flex-grow: 1;
+  display: flex;
+  gap: 16px;
+  align-items: center;
+  text-align: center;
+  padding: 0.625rem;
+  border: 2px solid transparent;
+  transition: all 300ms ease-in-out;
+  background: #3c3f4665;
+  width: 95%;
+  min-width: 256px;
+  max-width: 448px;
+}
+
+.main-program__card:hover {
+  border: 2px solid var(--white);
+  transform: scale(1.03);
+}
+
+.main-program__card__icon {
+  width: 3rem;
+}
+
+.main_program__card__header {
+  color: var(--primary-color-400);
+  text-align: left;
+  min-width: 80px;
+  max-width: 80px;
+}
+
+.main-program__card__description {
+  text-align: left;
+}
+
+/* Main Program Section Styles (Desktop) */
+@media (min-width: 40rem) {
+  .main-program {
+    background-size: 2vw;
+    padding: 6rem 2rem;
+  }
+
+  .main-program__headline {
+    margin-bottom: 3rem;
+    padding-top: 0;
+    margin-top: 0;
+  }
+
+  .main-program__cards-wrapper {
+    gap: 1.25rem;
+  }
+
+  .main-program__card {
+    flex-direction: column;
+    gap: 1rem;
+    width: 100%;
+    max-width: 192px;
+    padding: 16px;
+  }
+
+  .main-program__card__icon {
+    height: 3rem;
+    width: auto;
+  }
+
+  .main_program__card__header {
+    text-align: center;
+  }
+
+  .main-program__card__description {
+    text-align: center;
+  }
+}
+
+/* ============================================================ */
+
+/* Speakers Section Styles */
+.speakers {
+  background-color: var(--gray-400);
+  color: var(--gray-700);
+  padding: 4rem 0;
+}
+
+.speakers__headline {
+  font-size: var(--fs-700);
+  text-align: center;
+  margin-bottom: 48px;
+}
+
+.speakers__content {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 3rem;
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [OPTIONAL] I think you can give your content a width of 90% of the parent's element to fit in the box 

| Your Design | Figma Design |
| :---:   | :---: | 
|<img width="376" alt="image" src="https://github.com/henokkhm/microverse-module-1-capstone-project/assets/73854999/3b367b6f-83c6-43b4-b7e1-17a2ce2623ab"> | <img width="454" alt="image" src="https://github.com/henokkhm/microverse-module-1-capstone-project/assets/73854999/5f96d6bd-3e18-4240-acec-e69f9af04a73">|